docs/blog/2023/05/making-kafka-governance-breeze-for-developers.rst

```diff
@@ -0,0 +1,126 @@
+
+Making Kafka Governance a Breeze for Developers
+===============================================
+
+.. post:: May 15, 2023
+   :tags: kafka,governance,topics,acls,klaw
+   :category: kafka
+   :author: Murali Basani
+
+Background
+----------
+
+Kafka is widely adopted around the world due to its scalability and unique features that set it apart from traditional
+messaging. Many organizations are now using Kafka as part of their technology stack.
+
+.. code-block:: bash
+
+    A common challenge that companies face is inability to govern the hundreds
+    or even thousands of topics that are created and managing their access
+    permissions.
+
+
+This can be a difficult task, and companies often struggle to do it well.
+
+Some important considerations when managing Kafka topics include:
+
+- Who is allowed to produce and consume messages on a given topic and who owns it?
+- How do you backup and restore Kafka's configuration in case of failure?
+- Is security properly enforced through subscriptions and access controls?
+- How to smoothly promote topics with tested configurations from lower environments to production?
+- How to increase visibility and awareness of topics within the organization?
+
+
+While it is manageable to have a couple of hundred kafka topics, but as they keep growing together with applications
+and teams, it would certainly become hard to manage. There could be a need to isolate a bunch of topics from other teams.
+
```

Comment:
While a small number of topics may be manageable, as the use of Kafka grows along side additional applications and teams, it becomes harder to manage effectively and efficiently. Along with additional considerations of isolating sensitive topics that may contain GDPR, HIPPA or other sensitive data to only the applications that should have access to them.